web-console/script/create-sql-docs.js

```diff
@@ -53,17 +57,17 @@ const readDoc = async () => {
     }
   }
 
-  // Make sure there are at least 10 functions for sanity
-  if (functionDocs.length < 10) {
+  // Make sure there are enough functions found
+  if (functionDocs.length < EXPECTED_NUMBER_OF_FUNCTIONS) {
```

Comment:
Why not an equals check instead? Should the constant be renamed to `MINIMUM_EXPECTED_NUMBER_OF_FUNCTIONS`

Reply:
the constant should be renamed. I don't want someone adding a few SQL function to have to update this. If the SQL docs format change then this will still catch it because the number will drop from 130s to like 15